npath.go

```diff
@@ -0,0 +1,278 @@
+package tools
+
+import (
+	"fmt"
+
+	"github.com/bblfsh/sdk/uast"
+)
+
+//CodeReference
+//https://pmd.github.io/pmd-5.7.0/pmd-java/xref/net/sourceforge/pmd/lang/java/rule/codesize/NPathComplexityRule.html
+
```

Comment:
Use an appropiate godoc comment here. For example:

```
Npath computes the NPath of functions in a *uast.Node.

PMD is considered the reference implementation to assert correctness.
See: https://pmd.github.io/pmd-5.7.0/pmd-java/xref/net/sourceforge/pmd/lang/java/rule/codesize/NPathComplexityRule.html
```

Also, rename `Npath` to `NPath` everywhere.